### ISSUE TITLE
 torch.rfft(onesided=False) is deprecated, what should be the alternative?

### ISSUE THREAD

**Issue by ferric123** [mu-cai/frequency-domain-image-translation#1]
In latest pytorch versions, torch.rfft(onesided=False) is no longer available.
I tried both torch.fft.fft(image, 2) and torch.fft.rfft(image, 2). However, loss = criterion_L1(fake_fft * mask, real_fft * mask) in line 34 of utils_freq/freq_fourier_loss.py gives error, because fake_fft and real_fft have size (batch, 256) whereas mask has size (batch, 256, 256).
Could you please suggest an alternative for torch.rfft(onesided=False)?

**Comment by mu-cai** [comment]
Hi, we recommend using the older version of PyTorch, which could avoid such issues.
